tests/jsonnet/NOTICE

```diff
@@ -0,0 +1 @@
+Many of these tests are taken (and then modified for our purposes) from [go-jsonnet](https://github.com/google/go-jsonnet/)
```

Comment:
Do you know the percentage of the tests we took from go-jsonnet? Is it 100% or 50%?
And why we don't have all of them (given anyway we have eval_pass and eval_fail).